cypress/support/worldServices.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
You need to delete either `cypress/support/worldServices.js` or `cypress/support/serviceConfig.js` as theyre duplicate, tbh I prefer `cypress/support/worldServices.js` until a future refactor